Add bash completions from openindiana-completions project.
Remove pkgutil which we don't have.

--- bash-completion-2.16.0/completions/Makefile.am.orig
+++ bash-completion-2.16.0/completions/Makefile.am
@@ -510,7 +510,15 @@
 		zopfli \
 		zopflipng
 
-solaris_extra = pkgutil
+solaris_extra = \
+		beadm \
+		bootadm \
+		dladm \
+		pfexec \
+		pkg \
+		zfs \
+		zones \
+		zpool
 
 bsd_extra = pkg_delete
 
